tests/tests.mk

```diff
@@ -0,0 +1,94 @@
+ISTIO_GO := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+export ISTIO_GO
+SHELL := /bin/bash -o pipefail
+
+VERSION ?= 1.6-dev
```

Comment:
nit: `latest` may make more sense, so we don't need to update this with each branch